lib/ui/compositing.dart

```diff
@@ -272,10 +272,10 @@ class SceneHost extends NativeFieldWrapperClass2 {
   /// to the child.
   ///
   /// The scene host takes ownership of the provided export token handle.
-  SceneHost(int export_token_handle) {
+  SceneHost(dynamic export_token_handle) {
```

Comment:
Ok well we should definitely add documentation before going too far down this road.

But in the meantime, can you give a more detailed explanation? How is `dart:fidl.internal` defined, where is it exposed, how is the user expected to import this, how will they use it, what would code that uses this look like, etc?